gitlab_runner_config.py

```diff
@@ -314,7 +314,7 @@ def generate_runner_config(prefix, instance, tag_schema=None):
     except FileNotFoundError:
             schema = None
     try:
-        tagcap = importlib.import_module(p)
+        tagcap = importlib.import_module(args.capture_tags)
```

Comment:
Hi @tgmachina , sorry to ping you again! When you get the chance, could you take a look at this again? 
I have added the dictionary flattening, and moved scheduler tag generation. Of most interest is the method I am using to import a user-provided module.
Should the default values for the tag schema and capture tags values be our included files, or `None` ?

Reply:
That's a good question! I think that for the `tag_schema` the default should be `None`: no schema will be enforced by default.

As for the `capture_tags` method, I could go either way, but I think it would be fine to provide ours as a default.